CMake does not add MAGMA flags in standalone mode

When building Kokkos Kernels with MAGMA TPL, CMake does not add MAGMA flags to ```CMAKE_CXX_FLAGS```.  The statement ```IF (KOKKOSKERNELS_HAS_TRILINOS)``` in 
https://github.com/kokkos/kokkos-kernels/blob/9c5bbf914eb70bd3d3a6a51eff16615abf7c8d30/cmake/kokkoskernels_tpls.cmake#L453-L463
should be removed to make it work.

@e10harvey , Sorry I forgot to provide it
```
[vqdang@white28 kokkoskernels-build-white-magma]$ cmake ~/Kokkos/kokkos-kernels -DCMAKE_CXX_COMPILER=/home/vqdang/Kokkos/kokkos/bin/nvcc_wrapper -DCMAKE_INSTALL_PREFIX=~/Kokkos/kokkoskernels-install-white-magma -DCMAKE_BUILD_TYPE:STRING=RELEASE -DKokkos_DIR=~/Kokkos/kokkos-install-white/lib64/cmake/Kokkos -DKokkosKernels_INST_COMPLEX_DOUBLE:BOOL=ON -DKokkosKernels_ENABLE_TPL_MAGMA:BOOL=ON -DKokkosKernels_MAGMA_ROOT:PATH=$HOME/Packages/magma

[vqdang@white28 kokkoskernels-build-white-magma]$$ make -j install
```
Error message:
```
In file included from /ascldap/users/vqdang/Packages/magma/include/magma_lapack.h:14:0,
                 from /ascldap/users/vqdang/Kokkos/kokkos-kernels/src/impl/tpls/KokkosBlas_tpl_spec.hpp:70,
                 from /ascldap/users/vqdang/Kokkos/kokkos-kernels/src/impl/tpls/KokkosBlas_gesv_tpl_spec_decl.hpp:264,
                 from /ascldap/users/vqdang/Kokkos/kokkos-kernels/src/blas/impl/KokkosBlas_gesv_spec.hpp:162,
                 from /ascldap/users/vqdang/Kokkos/kokkoskernels-build-white-magma/src/impl/generated_specializations_cpp/gesv/Blas_gesv_eti_DOUBLE_LAYOUTLEFT_EXECSPACE_OPENMP_MEMSPACE_HOSTSPACE.cpp:48:
/ascldap/users/vqdang/Packages/magma/include/magma_mangling.h:32:10: error: #error "One of ADD_, NOCHANGE, or UPCASE must be defined to set how Fortran functions are name mangled. For example, in MAGMA, add -DADD_ to CFLAGS, FFLAGS, etc. in make.inc. If using CMake, it defines MAGMA_GLOBAL instead."
         #error "One of ADD_, NOCHANGE, or UPCASE must be defined to set how Fortran functions are name mangled. For example, in MAGMA, add -DADD_ to CFLAGS, FFLAGS, etc. in make.inc. If using CMake, it defines MAGMA_GLOBAL instead."
          ^~~~~
```

--- a/cmake/kokkoskernels_tpls.cmake
+++ b/cmake/kokkoskernels_tpls.cmake
@@ -450,7 +450,6 @@ KOKKOSKERNELS_ADD_TPL_OPTION(CUSPARSE ${CUSPARSE_DEFAULT}   "Whether to enable C
   DEFAULT_DOCSTRING "ON if CUDA-enabled Kokkos, otherwise OFF")
 
 IF (KOKKOSKERNELS_ENABLE_TPL_MAGMA)
-  IF (KOKKOSKERNELS_HAS_TRILINOS)
     IF (F77_BLAS_MANGLE STREQUAL "(name,NAME) name ## _")
       SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DADD_ -fopenmp -lgfortran")
     ELSEIF (F77_BLAS_MANGLE STREQUAL "(name,NAME) NAME")
@@ -460,7 +459,6 @@ IF (KOKKOSKERNELS_ENABLE_TPL_MAGMA)
     ELSE ()
       MESSAGE(FATAL_ERROR "F77_BLAS_MANGLE ${F77_BLAS_MANGLE} detected while MAGMA only accepts Fortran mangling that is one of single underscore (-DADD_), uppercase (-DUPCASE), and no change (-DNOCHANGE)")
     ENDIF()
-  ENDIF()
   LIST(APPEND TPL_LIST "MAGMA")
 ENDIF()
 

